src/tasks/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -121,7 +121,7 @@ public class AppleAppBuilderTask : Task
     public override bool Execute()
     {
         Utils.Logger = Log;
-        bool isDevice = Arch.Equals("arm64", StringComparison.InvariantCultureIgnoreCase);
+        bool isDevice = Arch.Equals("arm64", StringComparison.InvariantCultureIgnoreCase) && TargetOS != TargetNames.MacCatalyst;
```

Comment:
this is fine for now but I think we'll eventually need to split this into two variables: one to tell whether we require (Full)AOT and one whether we require signing with a developer account. As far as I know signing is required for arm64 MacCatalyst apps if you want to run it on a different machine.

Reply:
Plausible. I was mainly trying to avoid the FullAOT requirement